lib/modules/apostrophe-schemas/src/apos/components/ApostropheFieldMixin.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  props: {
+    value: Object,
+    field: Object
+  },
+  data() {
+    return {
+      next: (this.value.data !== undefined) ? this.value.data : (this.field.def || ''),
+      error: false
+    };
+  },
+  mounted() {
+    this.updateErrorAndEmit();
+  },
+  watch: {
+    next(value) {
+      this.updateErrorAndEmit();
+    }
+  },
+  methods: {
+    updateErrorAndEmit() {
+      const value = this.validate();
```

Comment:
this.validate does not return anything, do not modify value here... you will just pass on this.value... this.validate sets this.error.

Maybe it should return an error (or false) and you should assign it to this.error here though.